src/patterns/dosdp-patterns/cancer predisposition

```diff
@@ -0,0 +1,46 @@
+pattern_name: cancer_predisposition
+
+pattern_iri: http://purl.obolibrary.org/obo/mondo/patterns/cancer_predisposition.yaml
+
+description: 'This pattern should be used for terms in which hereditary cancer predisposition due to variation(s) in a gene confers the predisposition to a cancer.  
+
+contributors:
+- https://orcid.org/0000-0002-7638-4659
+
+classes:
+  cancer or benign tumor: MONDO:0700096
+  hereditary neoplastic syndrome: MONDO:0015356
+  gene: SO:0000704
+
+relations:
+  has material basis in germline mutation in: RO:0004003
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+  related_synonym: oio:hasRelatedSynonym
+
+vars:
+  gene: '''gene'''
+  disease: '''cancer or benign tumor'''
+  
+name:
+  text: '%s -related cancer predisposition'
+  vars:
+  - gene
+
+annotations:
+- annotationProperty: exact_synonym
+  text: tumor susceptibility linked to germline %s mutation
+  vars:
+  - gene
+
+def:
+  text: Hereditary cancer predisposition due to variation (s) in %s, which confers the predisposition to a %s.
+  vars:
+  - gene
+  - cancer
```

Comment:
"cancer" is not defined as a variable. you named the variable "disease" in line 24